src/book/03-guides/04-general/03-robocup-setup.mdx

```diff
@@ -0,0 +1,179 @@
+| ------------- | ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Edit          | <kbd>ENTER</kbd> | Edit the field currently highlighted. Press <kbd>ENTER</kbd> again when finished.                                                                                                                                                                                                        |
+| Toggle        | <kbd>SPACE</kbd> | Toggle between pre-determined fields, where applicable.                                                                                                                                                                                                                                  |
+| Refresh       | <kbd>F</kbd>     | Refreshes the ncurses UI.                                                                                                                                                                                                                                                                |
+| Reset         | <kbd>R</kbd>     | Resets to the current values on the system. Useful if you haven't saved yet and want to start over.                                                                                                                                                                                      |
+| Configure     | <kbd>C</kbd>     | Writes to the configuration files.                                                                                                                                                                                                                                                       |
+| Network       | <kbd>N</kbd>     | Writes to the configuration files and restarts the network services.                                                                                                                                                                                                                     |
+| Start RoboCup | <kbd>S</kbd>     | Starts the RoboCup service, which runs the `robocup` binary, including on restart.                                                                                                                                                                                                       |
+| Stop RoboCup  | <kbd>D</kbd>     | Stops the RoboCup service, and should be done when a robot is finished with a game. If this is not done then the next time someone uses the robot it will move. This can also cause robots out of the game to interfere with other robots behaviour due to robot-to-robot communication. |
```

Comment:
This sounds like something we could automate. I think systemd should have a thing for this so that when we ctrl c and exit the binary manually, the robocup service is turned off as well. From a quick search it seems that KillMode=control-group with ExecStopPost=systemctl disable robocup.service is the thing for this? @Bidski is what Im yapping about in the right track though? This comment is just a note for me or anyone who wants to implement this change

Reply:
There was also the idea (I think from Dex) about having a USB that controlled the service being on/off.

Reply:
IMO systemd solution leaves us an extra port to use which is slightly as it doesnt rely on a phys device

Reply:
I'm not sure how it would help though, we don't have any ssh connection to the robot during a game to stop the binary manually. It wouldn't solve the issue of the robot moving when it turns on, as the problem is that people forget to stop it before turning the robot off after a game or when doing repairs. With a USB, you can see that it's in and unplug it before the robot turns on.